src/core/meshcop/commissioner.cpp

```diff
@@ -91,16 +91,16 @@ otInstance *Commissioner::GetInstance(void)
 
 void Commissioner::AddCoapResources(void)
 {
-    mNetif.GetCoapServer().AddResource(mRelayReceive);
-    mNetif.GetCoapServer().AddResource(mDatasetChanged);
-    mNetif.GetSecureCoapServer().AddResource(mJoinerFinalize);
+    mNetif.GetCoap().AddResource(mRelayReceive);
+    mNetif.GetCoap().AddResource(mDatasetChanged);
+    mNetif.GetCoapSecure().AddResource(mJoinerFinalize);
 }
 
 void Commissioner::RemoveCoapResources(void)
 {
-    mNetif.GetCoapServer().RemoveResource(mRelayReceive);
-    mNetif.GetCoapServer().AddResource(mDatasetChanged);
-    mNetif.GetSecureCoapServer().AddResource(mJoinerFinalize);
+    mNetif.GetCoap().RemoveResource(mRelayReceive);
+    mNetif.GetCoap().AddResource(mDatasetChanged);
```

Comment:
Not directly related to the CoAP implementation, but shouldn't resources be removed here?